00.xsh

```diff
@@ -3,10 +3,8 @@ import time
 cd ../cf-graph
 $PATH.insert(0, '~/mc/bin')
 
-stages = [3, 0, 1, 2, 3]
+stage = 0
 start = time.time()
-for i in stages:
```

Comment:
What was the issue with this loop?

Reply:
Nothing, but I was setting it to run only one stage for each worker. Maybe I should leave the loop but change the `stages` list to only have one element in each script.

Reply:
Yea is rather just cut the list down since this is hopefully a temporary debug change.

Reply:
I think I will close this PR and open a new one with the changes since everything in this commit is going to be undone anyway. Let me know if you agree or if you think I should just add commits here.

Reply:
Also I realized make-cf-graph runs 03.xsh and cf-auto-tick runs 01.xsh, but I think those should be swapped.

Reply:
Whichever you prefer.